data/Prashanth-bokkala.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Prashanth Bokkala",
+  "displayStatusPublic": true,
```

Comment:
```suggestion
```
displayStatusPublic isnot required

Reply:
Yes , but it's doesn't problem for the code & they shown on the tutorial that displayStatusPublic .